ReactAndroid/src/main/java/com/facebook/react/animated/NativeAnimatedNodesManager.java

```diff
@@ -51,6 +52,9 @@
  */
 /*package*/ class NativeAnimatedNodesManager implements EventDispatcherListener {
 
+  // Used to avoid allocating a new array on every frame in runUpdates.
+  private final static List<AnimatedNode> sRunUpdateNodeList = new ArrayList<>();
```

Comment:
can you make this an instance variable unless it truly needs to be static and shared across NodesManagers?